.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord CIL community
+    url: https://discord.com/invite/9NTWu9MEGq
+    about: Contact our developers and fellow users in the Discord CIL community. 
+  - name: Contact us
+    url: https://ccpi.ac.uk/cil/
+    about: Contact us information in our landing page.
```

Comment:
hacky, but... add links to "create new PR" and "create new PR (internal/dev)" here.

Reply:
done, although the links are not ready yet. we need to merge the PR templates first